packages/e2e-tests/tests/object_preview-01.test.ts

```diff
@@ -41,7 +41,7 @@ test(`object_preview-01: expressions in the console after time warping`, async (
 
   await verifyConsoleMessageObjectContents(page, "Proxy{}", [
     "<target>: {a: 0}",
-    "<handler>: {get: ƒ()}",
+    "<handler>: {get: ƒget(target, prop, receiver)}",
```

Comment:
I didn't check - I assumed that this was some additional detail being displayed because the object now has a preview. I'll check this now (and also the remaining test failures).